Microsoft.Toolkit.Uwp.UI.Controls/WrapPanel/WrapPanel.cs

```diff
@@ -100,14 +100,16 @@ protected override Size MeasureOverride(Size availableSize)
             totalMeasure.U = Math.Max(lineMeasure.U, totalMeasure.U);
             totalMeasure.V += lineMeasure.V;
 
+            totalMeasure.U = Math.Ceiling(totalMeasure.U);
+
```

Comment:
Just use `Ceiling` here once, all problem solved.

How did I come out with that complex idea...